src/renderer/services/auth/types.ts

```diff
@@ -10,6 +12,8 @@ export interface AuthService {
   init(): Promise<AuthUser | null>;
   login(args: { email: string; password: string }): Promise<AuthUser | null>;
   logout(): Promise<void>;
+  reloadUser(): Promise<AuthUser | null>;
+  verifyEmail(): Promise<void>;
```

Comment:
```suggestion
  refreshUser(): Promise<void>;
  sendVerificationEmail(): Promise<void>;
```
Can we rename these methods? I feel like these method names are a bit more indicative of what the function should do.

Rather than returning the `AuthUser | null` object in `refreshUser` I would prefer us just notify the listeners of the new user object using the observable subject. See the relevant comment in `auth.service.ts`.